src/serac/infrastructure/tests/debug_print.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2019-2024, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include "axom/slic/core/SimpleLogger.hpp"
+#include <gtest/gtest.h>
+
+#include "serac/infrastructure/debug_print.hpp"
+#include "serac/infrastructure/application_manager.hpp"
+#include "serac/serac_config.hpp"
+
+namespace serac {
+
+TEST(DebugPrint, typeString)
+{
```

Comment:
would it be worth testing a more complex templated object? i guess the downside of that is then this test would need to depend on serac physics cmake targets.